pkg/ddc/vineyard/runtime_info.go

```diff
@@ -26,6 +26,11 @@ func (e *VineyardEngine) getRuntimeInfo() (base.RuntimeInfoInterface, error) {
 			return e.runtimeInfo, err
 		}
 
+		// Add the symlink method to the vineyard runtime metadata
+		runtime.ObjectMeta.Annotations = map[string]string{
```

Comment:
If there are other annotations in ObjectMeta, will they be overwrited?

Reply:
Thanks for pointing it out. I have changed the logic to avoid overwriting and added the unit test.